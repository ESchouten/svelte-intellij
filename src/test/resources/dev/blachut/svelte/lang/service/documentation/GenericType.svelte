<script lang="ts">
  class Box<T> {
    constructor(public value: T) {
    }
  }

  /**
   * docs have to be removed!
   */
  let local = new Box(323 as number | string);
</script>

{<caret>local.value}