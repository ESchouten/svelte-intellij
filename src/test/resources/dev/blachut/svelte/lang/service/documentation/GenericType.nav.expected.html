let local: <a href="psi_element://Box">Box</a>&lt;string | number&gt;<br/>[ide]<br/>let local: <a href="psi_element://Box">Box</a>&lt;number | string&gt;